packages/components/src/styles/interactive/interactive.stories.ts

```diff
@@ -64,88 +36,43 @@ export const Default = {
+  render: () => html`
+    <div class="flex flex-row gap-12">
+      <button type="button" class="sd-interactive">Lorem Ipsum</button>
+      <button class="sd-interactive sd-interactive--reset">Lorem Ipsum</button>
+    </div>
+  `
 };
 
 /**
- * Use the `inverted` class to invert the colors of an interactive element.
+ * Use the `sd-interactive--inverted` class when displayed on primary background.
+ *
+ * On darker backgrounds text/white inverted colour will substitute primary.
  */
 
 export const Inverted = {
-  parameters: { controls: { exclude: ['sd-interactive--inverted', 'sd-interactive--reset'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: [{ type: 'attribute', name: 'sd-interactive--inverted', values: [false, true] }]
-      },
-      options: {
-        templateContent: '<button class="%CLASSES%">%SLOT%</button>',
-        templateBackgrounds: { alternate: 'y', colors: ['white', 'rgb(var(--sd-color-primary, 0 53 142))'] }
-      },
-      constants: { type: 'attribute', name: 'sd-interactive--inverted', value: true },
-      args
-    });
-  }
+  render: () => html`
+    <div class="bg-primary p-4">
+      <button class="sd-interactive sd-interactive--inverted sd-interactive--reset">Lorem Ipsum</button>
+    </div>
+  `
 };
 
 /**
- * Use the `reset` class to reset the default browser styles of e. g. a button.
+ * Use the `sd-interactive--disabled` class to make an interactive element look disabled.
+ *
+ * This works as well when setting an `disabled` attribute on the element.
  */
 
-export const Reset = {
-  parameters: { controls: { exclude: ['sd-interactive--reset'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: [{ type: 'attribute', name: 'sd-interactive--reset', values: [false, true] }]
-      },
-      options: { templateContent: '<button class="%CLASSES%">%SLOT%</button>' },
-      constants: { type: 'attribute', name: 'sd-interactive--reset', value: true },
-      args
-    });
-  }
-};
-
-/**
- * Examples: This shows how sd-interactive looks in different contexts.
- */
-
-export const Examples = {
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'slot',
-          name: 'default',
-          values: [
-            'Text',
-            '<span>Text</span>',
-            '<sd-icon library="global-resources" name="system/picture"></sd-icon>'
-          ].map(content => {
-            return {
-              value: content,
-              title: content.replace('library="global-resources" name="system/picture"', '...')
-            };
-          }),
-          title: 'content'
-        }
-      },
-      options: { templateContent: '<button class="%CLASSES%">%SLOT%</button>' },
-      args
-    });
-  }
+export const Disabled = {
+  render: () => html`
+    <div class="flex flex-row gap-12">
+      <button class="sd-interactive sd-interactive--reset">Lorem Ipsum</button>
```

Comment:
This example can be removed and we only show the disabled one

Reply:
Updated